src/se/vidstige/jadb/managers/Package.java

```diff
@@ -14,7 +14,16 @@ public Package(String name) {
     public String toString() { return name; }
 
     @Override
-    public boolean equals(Object o) { return name.equals(o); }
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+
+        Package that = (Package) o;
+
+        if (!name.equals(that.name)) return false;
+
+        return true;
+        }
```

Comment:
Only very few parts of this function is tested, I think thats why codecov bails out. Are all these checks really needed? Specifically, the check for class equality will cause problems in case someone subclasses `Package`, right? Also the last two statements can just be written as 

    return name.equals(that.name);

To avoid a branch that would otherwise need to be tested.

Reply:
Also on line 26 there is a bit too much whitespace. I think just dropping this change is the easiest way to get the tests to pass.